tests/system/__snapshots__/system.spec.ts.snap

```diff
@@ -0,0 +1,92 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App Protocol Should fail on invalid method 1`] = `
+Object {
+  "error": Object {
+    "message": "Can not PUT /v1/test",
+  },
+  "status": "error",
+}
+`;
+
+exports[`App Protocol Should fail on malformed body 1`] = `
+Object {
+  "error": Object {
+    "message": "Unexpected end of JSON input",
+  },
+  "status": "error",
+}
+`;
+
+exports[`App Protocol Should fail when missing content type header 1`] = `
+Object {
+  "error": Object {
+    "fields": Object {
+      "key": Array [
+        "Required",
+      ],
+    },
+    "message": "key: Required",
+  },
+  "status": "error",
+}
+`;
+
+exports[`App Validation Should fail on handler input type mismatch 1`] = `
+Object {
+  "error": Object {
+    "fields": Object {
+      "something": Array [
+        "Expected string, received number",
+      ],
+    },
+    "message": "something: Expected string, received number",
+  },
+  "status": "error",
+}
+`;
+
+exports[`App Validation Should fail on handler output type mismatch 1`] = `
+Object {
+  "error": Object {
+    "fields": Object {
+      "anything": Array [
+        "Value should be greater than 0",
+      ],
+      "output": Array [
+        "Invalid format",
+      ],
+    },
```

Comment:
I'm not happy with it.
It seems that `.flatten()` takes only the first item from the path for making fields in `fieldErrors`.

Reply:
fixed in 90a711264a06ff6a8bf38b5cba3187def2ca7323